cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -626,13 +574,15 @@ struct ConvertFunctor {
                                                       parse_options_view const& opts,
                                                       bool as_hex)
   {
+    // TODO what's invalid input
```

Comment:
Is this TODO intended to be a question of what we should consider invalid?

Reply:
If this involves changes to fixed-point handling and isn't strictly related to trimming quotes, let's revert the changes here and file an issue or defer to another PR.

Reply:
I will split the changes to multiple PRs. There are 3 fixes in this PR.

Reply:
moved to https://github.com/rapidsai/cudf/pull/12022